### ISSUE TITLE
Now I am pretty sure this is a bug

### ISSUE THREAD

**Issue by batata004** [Semantic-Org/Semantic-UI#5663]
@Banandrew I am really sorry for reporting this as a bug and not going to the chat but now I am pretty sure this is a bug.

Check this minimal code -> https://www.quemfazsite.com.br/temp/teste666.php

I have 2 buttons exactly the same BUT the one in the right has also a ".search" class. For some reason the ".search" class makes the button to display incorrectly the arrow inside the first element. I only realized this problem when I used this .search button in conjunction with other buttons (that have no .search class) and the difference in layout was big. I use this exactly button in this site https://www.frasesparavida.com.br on the top right and if you click it you will also see the arrow is inside the first element.

![image](https://user-images.githubusercontent.com/8988923/29428411-7c3888f2-8363-11e7-8e82-766ac9429e44.png)

Is there any solution I can apply?

I also would like to ask you why some examples in the DROPDOWN link examples have arrows on top of the menu and others dont have that arrow? I cant find a specific class that makes that nice arrow to show up!

**Comment by Banandrew** [comment]
@batata004 Confirming the issue, I’ll take a look at it as soon as I can.

**Comment by batata004** [comment]
;) Thanks my friend. If somebody for example fix this issue today (I know nobody will do that today, this is an open community and things may take long), do you know how many days it will take to be in production/CDN available?

**Comment by Banandrew** [comment]
@batata004 No, probably a while, but I’ll post a temporary fix that you’ll be able to use.

**Comment by batata004** [comment]
\o/ again no words to thank you! I feel sorry for you cause you look like the only person caring about this library. Semantic UI is awesome, completely awesome. Why arent there so much people helping? I see lots of people involved in this project, it's very well populated with many people but most of them do absolutelly nothing. I only see you replying and taking care of the library.

**Comment by Banandrew** [comment]
@batata004 This cannot be quickly patched, because `.pointing` uses `::after` to show the pointer, and any kind of scrolling menus use `overflow` properties to show a scrollbar and limit the results, so the pointer becomes hidden—you’ll have to figure out a way to work around it at this time, the issue’s a bit complicated.

**Comment by batata004** [comment]
I thought it was as simple as adding a padding-top to the ::after element and it would all work fine, I didnt know it was that complicated. Anyway, I appreciate your attention, I will not bring this up anymore! When I have some spare time in this weekend I will try to take a look where I could add this padding-top to make this work easily. If I find a way I will get in touch ;)

**Comment by Banandrew** [comment]
For the future reference: the issue is very similar to #4446, both should probably be dealt with simultaneously.

**Comment by FelipeACP** [comment]
Did anyone find any workaround for that?
